QUARKUS-2394.md

```diff
@@ -0,0 +1,39 @@
+# QUARKUS-2394 Drop OpenTelemetry Jaeger exporter
+
+Jira: https://issues.redhat.com/browse/QUARKUS-2394
+
+Opentelemetry Jaeger exported has been deprecated from an official Quarkus support point of view. The extension `quarkus-opentelemetry-exporter-jaeger` will be moved to Quarkiverse and `quarkus-opentelemetry-exporter-otlp` extension has been promoted as the supported one. 
+
+## Scope of the testing
+
+All references to `quarkus-opentelemetry-exporter-jaeger` are going to be removed. Current coverage are going to be replaced by `quarkus-opentelemetry-exporter-otlp` on OpenShift / Baremetal / Native
+
+### Impact on test suites and testing automation
+
+The main impact are related to scenarios configuration and documentation in the following test suites:
+- Quarkus Test Suite
+- Beefy Scenarios
```

Comment:
Drop BeefyTS, it's not actively maintained. We can keep the legacy exporter there.

Reply:
well...the PR is already done :(

Reply:
Apologies.